tests/unit/test_vecpwconstant.py

```diff
@@ -179,8 +179,13 @@ def test_error_l2(self):
         # fmt: on
         ana_sol = lambda x: np.sin(x)
 
+        err = discr.error_l2(sd, num_sol, ana_sol, etype="standard")
+        err_known = 7.951651025069553e-08
+
+        self.assertTrue(np.allclose(err, err_known))
+
         err = discr.error_l2(sd, num_sol, ana_sol)
-        err_known = 5.622666361455642e-08
+        err_known = 0.5254595621486393
```

Comment:
Let's double check why this error is so large

Reply:
cause the grid is very coarse and the function we are evaluating is the sin, when refining the error decays as expected. for this l2-error the computation of the analytical solution is done on the vertices and compared with the one in the cell centers. while for the other evaluation of the error (etype="standard") the analytical solution is interpolated over the cell centers